examples/headsail-bsp/Cargo.toml

```diff
@@ -16,6 +16,7 @@ sysctrl-rt = [
 panic-uart = []
 hpc = []
 sysctrl = []
+allocator = ["dep:good_memory_allocator", "hpc"]
```

Comment:
This feature is typically called `alloc` by convention.

Reply:
All references to allocator changed to alloc